packages/docs/content/foundation/typography/list.mdx

```diff
@@ -2,7 +2,14 @@
 title: List
 order: 40
 intro: Lists organize written information for users sequentially.
-
+core:
```

Comment:
I couldn't find List in our docs navigation?

Reply:
https://design.cms.gov/foundation/typography/list/?theme=core